src/makers/generic/zip.js

```diff
@@ -4,6 +4,9 @@ import pify from 'pify';
 import zipFolder from 'zip-folder';
 
 import { ensureFile } from '../../util/ensure-output';
+import { checkSupportedPlatforms } from '../../util/check-supported-platforms';
```

Comment:
I am OK with breaking compatibility, assuming that we major version bump and label the breaking commit appropriately so that it's recognizable in the changelog.